ui/packages/catalog-ui-search/src/main/webapp/component/alert-item/alert-item.view.js

```diff
@@ -47,6 +48,13 @@ module.exports = Marionette.ItemView.extend({
   },
   removeModel() {
     this.$el.toggleClass('is-destroyed', true)
+    fetch('./internal/user/notifications', {
+      method: 'put',
+      body: JSON.stringify({ alerts: [this.model.id] }),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    })
     setTimeout(() => {
       this.model.collection.remove(this.model)
       user
```

Comment:
❓ Can we remove this `savePreferences` call now that alerts are no longer being stored in user prefs?

Reply:
Yes - it still saves preferences (including the alerts, which is useless to save) since we are updating the user model, which is undesirable. It gets pretty bad when clearing all notifications, since that removes each alert one by one. This used to trigger 6 preference updates to delete 6 alerts, now it triggers 6 (useless) preferences updates and 6 calls to `user/notifications`. But we can definitely delete this.